### PR TITLE
Resmod

### PR DESCRIPTION
**Pull request by ZheHuang96**
Hi Rikard,

Here are two commits for resmod. Mats and Xiaomei suggest comparing different significant levels in resmod, so I add an option to change the cutoffs by users. The other commit is to avoid unnecessary tasks when running resmod multiple times and make the resmod model results data frame flexible.

Best regards,
Zhe Huang

### CODE REVIEW

File: src/pharmpy/tools/resmod/tool.py [hunk 1]
```diff
@@ -35,26 +35,31 @@ def create_workflow(model=None, groups=4):
...
-        fit_wf, predecessors=[task_base_model, task_iiv, task_power, task_combined] + tasks
-    )
-
-    task_post_process = Task('post_process', post_process)
+    if 'IIV_on_RUV' not in skip:
+        task_iiv = Task('create_iiv_on_ruv_model', _create_iiv_on_ruv_model)
+        tasks.append(task_iiv)
+        wf.add_task(task_iiv, predecessors=task_base_model)
+
+    if 'power' and 'combined' not in skip:
```
Comment (rikardn) [inline src/pharmpy/tools/resmod/tool.py:44]: This is not what you intend. It will be testing `'combined' not in skip` only.

**Review by rikardn — CHANGES_REQUESTED** [review]
Just one error.

File: src/pharmpy/tools/resmod/tool.py (hunk 1 above)
Comment (ZheHuang96) [reply src/pharmpy/tools/resmod/tool.py]: Thank you for your comment! I forgot it...I correct the code now!

Comment (ZheHuang96) [reply src/pharmpy/tools/resmod/tool.py]: It should be `if 'power' not in skip and 'combined' not in skip:`

**Comment by rikardn** [comment]
LGTM

**Pull request merged**